texttospeech/snippets/audio_profile.go

```diff
@@ -0,0 +1,65 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package snippets contains speech examples.
+package snippets
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+
+	// [START imports]
```

Comment:
Our new snippet style has each snippet in a separate file then include the full import block in the region tag. Can we do that here?

Also, all region tags should be in the samples tracker, which requires the tag starts with the name of the product.

Reply:
Also, this tag doesn't include all of the necessary imports (e.g. `fmt`).

Reply:
Got it, updated.